translations/about.yml

```diff
@@ -4,7 +4,7 @@ messages:
     welcome-feature-generate: 'Genera cod șablon.'
     welcome-feature-interact: 'Interacționa cu o instanță de Drupal 8.'
     welcome-feature-learn: 'Învăța Drupal 8.'
-    links: 'Project links'
+    links: 'Linkuri de proiect'
```

Comment:
Aici am impresia ca "The" e implicit, si se doreste exprimarea posesiei, nu a tipului de link-uri. Deci Link-urile proiectului.